exe-unit/components/gsb-http-proxy/tests/test_proxy.rs

```diff
@@ -0,0 +1,98 @@
+use actix_http::header::HeaderMap;
+use actix_web::{web, HttpResponse, Responder};
+use actix_web::{App, HttpServer};
+use chrono::Utc;
+use futures::StreamExt;
+use test_context::test_context;
+use ya_framework_basic::async_drop::DroppableTestContext;
+use ya_framework_basic::log::enable_logs;
+use ya_gsb_http_proxy::http_to_gsb::HttpToGsbProxy;
+use ya_gsb_http_proxy::message::GsbHttpCallMessage;
+use ya_gsb_http_proxy::response::GsbHttpCallResponseEvent;
+use ya_service_bus::typed as bus;
+
+#[test_context(DroppableTestContext)]
+#[serial_test::serial]
+pub async fn test_gsb_http_proxy(ctx: &mut DroppableTestContext) {
+    enable_logs(true);
```

Comment:
Should be disabled after development is finished
```suggestion
    enable_logs(false);
```